catalog/models.py

```diff
@@ -1,3 +1,74 @@
 from django.db import models
 
-# Create your models here.
+
+class Category(models.Model):
+
+    name = models.CharField(max_length=150, verbose_name="Наименование")
+    description = models.TextField(verbose_name="Описание")
+
+    def __str__(self):
+        return f"{self.name}\n {self.description}"
+
+    class Meta:
+        verbose_name = "категория"
+        verbose_name_plural = "категории"
+
+
+class Product(models.Model):
+
+    name = models.CharField(max_length=150, verbose_name="Наименование")
+    description = models.TextField(verbose_name="Описание")
+    image = models.ImageField(
+        upload_to="images/", verbose_name="Изображение", blank=True, null=True
+    )
+    category = models.ForeignKey(
+        Category,
+        on_delete=models.CASCADE,
+        verbose_name="Категория товара",
+        blank=True,
+        null=True,
+        related_name="products",
+    )
+    purchase_price = models.FloatField(verbose_name="Цена за покупку")
```

Comment:
Для цены корректнее использовать `DecimalField`, это `numeric` в Postgres - самый подходящий для денег, так как является более точным
```suggestion
    purchase_price = models.DecimalField(max_digits=10, decimal_places=2, verbose_name="Цена за покупку")
```